### DISCUSSION TITLE
post/payment-feed-server/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#104] [Announcements]
# post/payment-feed-server/

여러 MSA를 통해 콘텐츠를 서빙하는 결제탭 피드 서버에서 유연한 구조를 찾기 위해 시도한 코드 아키텍처를 공유합니다.

https://tech.kakaopay.com/post/payment-feed-server/

**Comment by LichKing-lee** [comment]
다양한 컨텐츠를 조합해야하는 상황에선 어떻게 해야할지 고민이 많았는데 많은 도움이 됐습니다 좋은 글 감사합니다!!
